.claude/scripts/polarion_client.py

```
@@ -0,0 +1,395 @@
+"""
```

Ruff (0.15.17)

[warning] The file is executable but no shebang is present

(EXE002)